polymer-redux.js

```diff
@@ -54,18 +53,23 @@
                     value = Polymer.Base.get(path, state);
                 }
 
+                // prevent unnecesary polymer notifications
+                previous = element.get(property.name);
+                if(value === previous) {
+                    return;
```

Comment:
I would agree with @cvanleeuwen that this looks like a good optimization. I'd consider it fair to assume that developers know at least the three principles of Redux (http://redux.js.org/docs/introduction/ThreePrinciples.html).